schema/schema.go

```diff
@@ -66,6 +79,29 @@ func Get(ctx context.Context, subject string, version int) (schema.Schema, error
 	return resp.Schema, nil
 }
 
+// -- Schema context utilities -------------------------------------------------
+
+type schemaContextNameKey struct{}
+
+func WithSchemaContextName(ctx context.Context, schemaCtxName string) context.Context {
```

Comment:
Can we add comments on these two methods, explain what they are used for and explicitly tell developers that they don't have to use them in connectors?